sherpa/plot/dummy_backend.py

```diff
@@ -36,14 +47,19 @@
            'get_histo_defaults', 'get_model_histo_defaults',
            'get_component_plot_defaults', 'get_component_histo_defaults',
            'vline', 'hline', 'get_scatter_plot_defaults',
-           'get_cdf_plot_defaults', 'get_latex_for_string', 'name')
+           'get_cdf_plot_defaults', 'get_latex_for_string', 'name',
+           'initialize_plot', 'select_plot')
 
 
 lgr = logging.getLogger(__name__)
 
 # Identify the backend
 name = 'dummy'
 
+warning = lgr.warning
+warning('Failed to import usable sherpa.plotting backends.' +
```

Comment:
```suggestion
lgr.warning('Failed to import usable sherpa.plotting backends.' +
```

Not exactly the most-important comment ever